src/docs/swagger.ts

```diff
@@ -34,7 +34,12 @@ const options: swaggerJsdoc.Options = {
       },
     },
   },
-  apis: ['./src/docs/documentation.ts', './src/docs/auth.docs.ts', './src/docs/user.docs.ts'],
+  apis: [
+    './src/docs/documentation.ts',
+    './src/docs/auth.docs.ts',
+    './src/docs/user.docs.ts',
+    './src/docs/admin.disable.account.docs.ts',
+  ],
```

Comment:
well done @Ke-bean, can you take a look at how martine handled this in her pr [here](https://github.com/atlp-rwanda/e-commerce-serge-30-bn/pull/33/files#diff-f12f94160b259265ab67db45f23d8da85aba229c2f7492ae38d2342e0b23bc2e) and see if you can change it like that as well in case her work isn't merged yet.

Reply:
> well done @Ke-bean, can you take a look at how martine handled this in her pr [here](https://github.com/atlp-rwanda/e-commerce-serge-30-bn/pull/33/files#diff-f12f94160b259265ab67db45f23d8da85aba229c2f7492ae38d2342e0b23bc2e) and see if you can change it like that as well in case her work isn't merged yet.

Done! @Oliviier-dev